src/simplewallet/simplewallet.cpp

```diff
@@ -4767,7 +4765,7 @@ bool simple_wallet::sweep_unmixable(const std::vector<std::string> &args_)
     }
     else if (m_wallet->watch_only())
     {
-      bool r = m_wallet->save_tx(ptx_vector, "unsigned_monero_tx");
+      bool r = m_wallet->save_unsigned_tx_to_file(ptx_vector, "unsigned_monero_tx");
```

Comment:
Are all these name changes arbitrary, or somehow needed ?

Reply:
I proposed those name changes which are not strictly needed, but I think they improve the general readability, given that there are quite a few functions for saving/loading unsigned txes and signing txes which read from string/file and and write to string/file. The functionality can be inferred from the argument's types and names, but I think the intent would become clearer if the kind of combination is reflected in the function name.

Do you think these name changes are gratuitous?